### PR TITLE
Add current PHP versions to Travis CI tests

### PR DESCRIPTION
**Pull request by pixelbrackets**
### Description:

Follow-Up for #71 - WIP

* Add current PHP versions (7.4, 8.0, 8.1)

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [x] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [x] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)

### CODE REVIEW

**Comment by pixelbrackets** [comment]
I splitted the PR, since newer PHP versions are not supported by the exitsting test framework. This needs additional work. So I would like to keep this open as work in progress, please merge #71 first.

**Comment by pixelbrackets** [comment]
Fixed the tests for newer versions. Ready to merge.

About the oleder PHP versions: I am very much in favour of this. I opened a discussion issue about dropping support for these versions (not just ignore them in tests) #72

File: .travis.yml [hunk 1]
```diff
@@ -3,15 +3,14 @@ language: php
...
     fast_finish: true
     include:
+      - php: 8.1
+      - php: 8.0
+      - php: 7.4
       - php: 7.3
       - php: 7.2
       - php: 7.1
       - php: 7.0
       - php: 5.6
```
[the remark this answers is not in the document]
Comment (pixelbrackets) [reply .travis.yml]: I am very much in favor of this. I opened a discussion issue about dropping support for these versions (not just ignoring them in tests) #72

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "license": "GPL-3.0+",
+    "authors": [
+        {
+            "name": "The Matomo Team",
+            "email": "hello@matomo.org",
+            "homepage": "https://matomo.org/team/"
+        }
+    ],
+    "require": {
+    },
```
[the remark this answers is not in the document]
Comment (pixelbrackets) [reply composer.json:14]: Agree. However, I would like to do this in a follow up PR, altogether with the other required changes mentioned in #72, and not wait for the necessary clarifications and approvals in this PR right here. Small steps. Agree?

[the remark this answers is not in the document]
Comment (sgiehl) [reply composer.json:14]: makes sense

**Review by sgiehl — APPROVED** [review]
Looks good. Btw. at some later point we should also consider to move from travis tests to github actions. We are also preparing this for our other repos. So if you are familiar with github actions, feel free to also create a PR for that.

**Pull request merged**